### ISSUE TITLE
OneClick Install Error

### ISSUE THREAD

**Issue by qlulezz** [Assistant/ModAssistant#40]
![UnrecognizedArgument-ClosingModAssistant](https://user-images.githubusercontent.com/54777265/64080471-c3c40200-ccf4-11e9-8fef-9d0252e939ef.png)

"Unrecognized argument. Closing Mod Assistant."
This message keeps popping up everytime I try to intall songs with OneClick on beatsaver.com (and modelsaber.com). I've had this problem for like half a year but I could't get it fixed since then. Kyle tried to help me on the Discord, but with no success and nobody else could help me since I'm the only one with this problem. A clean reinstall of Beat Saber didn't help either, now I am trying my luck here.. :)

**Comment by Assistant** [comment]
Try saving it somewhere other than the Beat Saber install folder, it has weird quirks when located there (and probably inside Program Files, too) for reasons beyond my comprehension.
I can't test it to see if I can fix it, but I could make some debugging builds to if you're willing to try them out.

**Comment by HenryCraft** [comment]
I tried it with the .exe on the Desktop and on another drive, but it didn't made a difference. Since the only way to install songs from beatsaver without putting the headset on is to start the game with fpfc and install it in the songbrowser (because the search funktion in BeatDrop bugs :/) I am going to test your debugging builds if you made them. :)

**Comment by Assistant** [comment]
You have to disable and re-enable the OneClick Install options to refresh the location, otherwise it just keeps using the one in the game folder. I'll make a debug build in a bit.

**Comment by Assistant** [comment]
Please try this, and show me what it says.
[ModAssistant.zip](https://github.com/Assistant/ModAssistant/files/3563904/ModAssistant.zip)

**Comment by HenryCraft** [comment]
![OneClickMessage1](https://user-images.githubusercontent.com/54777265/64082584-7c983a00-cd11-11e9-804f-8bb9c8618d99.png)

**Comment by HenryCraft** [comment]
Wait, IT WORKS!

![ITWORKS](https://user-images.githubusercontent.com/54777265/64082696-3a6ff800-cd13-11e9-89bc-662ef76c6ae8.png)

When clicking on OneClick Install on beatsaver or modelsaver it tells me that I have to choose an Application to open it with, now "modassistant" appeared in the list and when choosing it, it installs successfully. I don't know what I did that made it appear, but choosing the wrong application was the problem. Because "modassistant" didn't show up, I selected the ModAssistant.exe by myself and that made it not work properly.
